core/services/ocr2/plugins/ocr2keeper/evm/abi.go

```diff
@@ -3,17 +3,13 @@ package evm
 import (
 	"fmt"
 	"math/big"
-	"strings"
 
 	"github.com/ethereum/go-ethereum/accounts/abi"
-	"github.com/ethereum/go-ethereum/common"
 	"github.com/ethereum/go-ethereum/common/hexutil"
 	ocr2keepers "github.com/smartcontractkit/ocr2keepers/pkg"
-)
 
-type evmRegistryPackerV2_0 struct {
-	abi abi.ABI
-}
+	iregistry21 "github.com/smartcontractkit/chainlink/v2/core/gethwrappers/generated/i_keeper_registry_master_wrapper_2_1"
```

Comment:
i.e. to have 2.0 as `evm` and 2.1 as `evm21`? thought about it as well because there are less changes.

We said that because the final folder is called `evm` it be easier to work on the same files to avoid merging cross folders, but in general we can manage either way.

Reply:
oh i was recommending evm20 and evm21. No confusion as to what evm stands for :sweat_smile:

Reply:
Yeah, we can maybe go all the way and make it `evmregistry2x`